README.md

```diff
@@ -61,13 +61,15 @@ A curated list of awesome books, videos, courses and resources about making a st
   - [Do thing's that don't scale](http://paulgraham.com/ds.html)
   - [18 Mistakes that kill startups](http://www.paulgraham.com/startupmistakes.html)
   - [How to raise money](http://www.paulgraham.com/fr.html)
+ - [High Output Management - Andrew S. Grove (book)](https://www.amazon.de/High-Output-Management-Andrew-Grove/dp/0679762884)
```

Comment:
Thanks for the suggestion.
With that all the resources are free in this list do you thing a link to the book on amazon should fit?

Reply:
thanks. I saw bunch of amazon.com links that is why I added. Anyway, it is a really nice book and should be read by all managers/entrepreneurs.